pkg/handlers/pvchandler.go

```diff
@@ -101,7 +104,7 @@ func (pvcHandler *PvcHandler) enoughLvCapacity(pvc v1.PersistentVolumeClaim) boo
 func shouldPvcBeHandled(pvc v1.PersistentVolumeClaim, nodeName string, storagePath string) (bool, string) {
 	if pvc.Status.Phase == v1.ClaimPending {
 		if pvcNodeName, ok := pvc.ObjectMeta.Annotations[k8sclient.NodeName]; ok && (pvcNodeName == nodeName) {
-			pvDir := storagePath + "/" + pvc.ObjectMeta.Namespace + "_" + pvc.ObjectMeta.Name
+			pvDir := storagePath + "/" + pvc.ObjectMeta.Namespace + "_" + pvc.ObjectMeta.Name + "-" + strings.Split(string(pvc.ObjectMeta.UID), "-")[0]
```

Comment:
I recall we discussed not to draw dependency between PVC and volume, as it belongs to the PV
generate a unique identifier instead, and use that as the name of the PV!

Reply:
ok, fixed